src/SLCore.Listeners/SLCore.Listeners.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\ConnectedMode\ConnectedMode.csproj" />
+    <ProjectReference Include="..\IssueViz.Security\IssueViz.Security.csproj" />
```

Comment:
Seems like a very bad architectural decision to add a refernce to IssueViz.Security inside SlCore. Wouldn't it have been better to raise an event in the TaintVulnerabilityListener.cs and make the TaintMonitor listen to that event as well?

Reply:
Discussed offline: this assembly was created to be able to have non-dummy implementations for listeners, it is fine to add required references to it.